components/file_system/src/io_stats/mod.rs

```diff
@@ -0,0 +1,88 @@
+// Copyright 2022 TiKV Project Authors. Licensed under Apache-2.0.
+
+#[cfg(not(any(target_os = "linux", feature = "bcc-iosnoop")))]
```

Comment:
In my understanding, this is to replace `null.rs`?

Reply:
Yes. It's a stub of interfaces that each io_stats implementation must adhere to.